.github/styles/Mautic/FeatureList.yml

```diff
@@ -1,7 +1,7 @@
 extends: substitution
 message: "Is this referring to a Mautic feature? If so, use '%s' instead of '%s'."
 level: suggestion
-ignorecase: true
+ignorecase: false
```

Comment:
```suggestion
ignorecase: true
```

This should be true. We [capitalise proper nouns ](https://contribute.mautic.org/education-team/technical-writing-styleguide#style-guide-and-copy-rules)including features of Mautic to differentiate between the Mautic feature, and other uses of the word. For example Email (the thing in Mautic) rather than the act of sending email.

Reply:
Exactly, so ignoring the case seems wrong then, no? We should be comparing things while taking the case into account?